students/jsward/session02/grid_printer.py

```diff
@@ -0,0 +1,24 @@
+
+def print_grid(grid_size, grid_qty):
+    """
+    Generate a grid with specified cell size and cell quantity
+    :param grid_size: integer
+    :param grid_qty: integer
+    :return: none
+    """
+
+    total_size = ((grid_size +1) * grid_qty) + 1
+    i = 0
+    while i < total_size:
+        if i % (grid_size +1) == 0 or i == (total_size -1):
+            grid_square = "+{}".format('-' * grid_size)
+            print("{}+".format(grid_square * grid_qty))
+        else:
+            grid_square = "|{}".format(' ' * grid_size)
+            print("{}|".format(grid_square * grid_qty))
+        i += 1
+
+
+grid_size = input("Cell size: ")
```

Comment:
This is great, but it won't run if you try it. Don't forget to change your input from a String to an Int 😄

Reply:
so oddly, this works from the terminal in pycharm, but I did encounter the type error when I run the code with the debugger.  Added type conversion to fix this issue.

Reply:
That's suspicious -- is there a chance that PyCharm is using Python 2 when you run it?

Python 2 had a different `input()` function.